config/manager/manager.yaml

```diff
@@ -56,5 +49,5 @@ spec:
           requests:
             cpu: 10m
             memory: 64Mi
-      serviceAccountName: controller-manager
+      serviceAccountName: csi-addons-controller-manager
```

Comment:
Why is this not taken from the default anymore?

Reply:
> Why is this not taken from the default anymore?

kustomize does not seem to add prefix to `serviceAccountName` when built from config/manager/kustomize.yaml